app/services/customers/create_service.rb

```diff
@@ -187,6 +194,13 @@ def create(**args)
 
     private
 
+    def valid_email?(email)
+      return true if email.nil?
+
+      email_regex = /\A[\w+\-.]+@[a-z\d\-.]+\.[a-z]+\z/i
```

Comment:
Where the regex from?
I think we should copy the regex from Devise: https://github.com/heartcombo/devise/blob/72884642f5700439cc96ac560ee19a44af5a2d45/lib/devise.rb#L114-L118

Reply:
changed